src/function/table_functions/call_functions.cpp

```diff
@@ -12,6 +12,13 @@ using namespace kuzu::common;
 using namespace kuzu::catalog;
 using namespace kuzu::main;
 
+function_set CurrentSettingFunction::getFunctionSet() {
+    function_set functionSet;
+    functionSet.push_back(
+        std::make_unique<TableFunction>("current_setting", tableFunc, bindFunc, initSharedState));
+    return functionSet;
+}
```

Comment:
double check coverage

Reply:
I think it is a false positive. They are guaranteed to be covered